package.json

```diff
@@ -1,9 +1,12 @@
 {
   "name": "stellar-base",
-  "version": "10.0.0-soroban.3",
+  "version": "10.0.0-soroban.4",
   "description": "Low-level support library for the Stellar network.",
   "main": "./lib/index.js",
-  "browser": "./dist/stellar-base.min.js",
+  "browser": {
+    "main": "./dist/stellar-base.min.js",
```

Comment:
Hi @Shaptic, can it be correctly recognized by other tools? I noticed that stellar-sdk uses `./lib/index.js` instead of `./dist/stellar-base.min.js` when generating `stellar-sdk.js`. Is this the expected behavior?

Reply:
Additionally, for j`s-xdr`, the stellar-sdk uses `./dist/js-xdr.min.js`.